bridges/DealabsBridge.php

```diff
@@ -0,0 +1,134 @@
+<?php
+class DealabsBridge extends BridgeAbstract {
+	const NAME = 'Dealabs search bridge';
+	const URI = 'https://www.dealabs.com/';
+	const DESCRIPTION = 'Return the Dealabs search result using keywords, with/without expired deals, with/without shop deals and by category';
+	const MAINTAINER = 'sysadminstory';
+	const PARAMETERS = array( array (
+		'q' => array('name' => 'Mot(s) clé(s)', 'type' => 'text', 'required' => true ),
+		'expired_choice' => array('name' => 'Afficher deals expirés', 'type' => 'checkbox'),
+		'instore_choice' => array('name' => 'Afficher deals en magasin', 'type' => 'checkbox'),
```

Comment:
This part looks a bit off. Please indent it like the arrays below (one element per line)